### DISCUSSION TITLE
Migration ?

### DISCUSSION THREAD

**Discussion by jpmurray** [community-scripts/ProxmoxVE#484] [Q&A] [answered]
Anything I should be doing to migrate from the old repo to the community scripts in my current setup?

**Comment by havardthom** [comment, marked as answer]
Running this will fix links in LXCs from old repo: https://community-scripts.github.io/ProxmoxVE/scripts?id=update-repo

**Reply by jpmurray** [reply]
Thank you !

**Reply by jeeftor** [reply]
So after running this whats the process to update an LXC to a new version?

**Reply by michelroegl-brunner** [reply]
Just run update in the lxc or run the according script from the website in the lxc
